changehc/delphi_changehc/flag_data.py

```diff
@@ -0,0 +1,98 @@
+"""A file in every indicator pipeline to create the appropriate raw dfs."""
```

Comment:
Maybe we should hop on a call to all discuss this @krivard 
- it will create some duplicate code, but because we also need to be able to run the flagging system on raw counts (not just API), I think we need some kind of calling code that is indicator-specific. 
-

Reply:
From video chat Nat <> Ananya: The `flag_data` code added to `changehc` here generates unsmoothed daily data and saves it in the date x state input format that the flagging module wants.

`changehc` needs special handling here because, unlike most other signals, its main pipeline only creates 7-day average smoothed versions, despite unsmoothed values being available. The `flag_data` addendum makes those unsmoothed values available _without_ passing them through the API.

An alternative approach is to have the `changehc` pipeline report to the API the unsmoothed versions of its current signals. I'm not sure if there's a blocker here (privacy concerns?).

Most other indicators report unsmoothed signals to the API already and do not need this type of addition.

`dsew-cpr` does not report unsmoothed values because none are available. As a result `dsew-cpr` will not have flagging run on it.